ios/Classes/AudioServicePlugin.m

```diff
@@ -405,7 +405,9 @@ - (void) updateControl:(enum MediaAction)action {
             if (rewindInterval.integerValue > 0) {
                 if (enable) {
                     [commandCenter.skipBackwardCommand addTarget: self action:@selector(skipBackward:)];
-                    commandCenter.skipBackwardCommand.preferredIntervals = @[rewindInterval];
+                    int rewindIntervalInSeconds = (int) [rewindInterval floatValue]/1000;
```

Comment:
Is `floatValue` correct for an integer? And I note that the (int) typecast applies only to the numerator so it's converting back to an integer anyway. But then you're dividing by /1000 which converts it back to a double, and then there's an implicit conversion to int when assigning it to an int variable. Not sure if this was intentional, but it seems like a roundabout way to get the result.

Reply:
you might be right. I am not very familiar with objective C. So I probably did too much for a simple division operation. Maybe then, I can just divide [rewindInterval intValue] by 1000 and then it implicitly converts the whole value to int. Is that correct? If so do you want me to do it?

Reply:
I'm also no expert when it comes to this peculiar language. But as long as it works and is straightforward, I can use it. At the very least, I would avoid the redundant back-and-forth type conversions.